source/Calamari.Scripting/Behaviours/SubstituteScriptSourceBehaviour.cs

```diff
@@ -26,7 +26,7 @@ public bool IsEnabled(RunningDeployment context)
 
         public Task Execute(RunningDeployment context)
         {
-            substituteInFiles.Substitute(context, ScriptFileTargetFactory(context).ToList());
+            substituteInFiles.Substitute(context.CurrentDirectory, ScriptFileTargetFactory(context).ToList());
```

Comment:
Updated since the method was updated here https://github.com/OctopusDeploy/Calamari/commit/139c46c7959a88f79ae6e82dc3fd6becf6c10ab4#diff-56bcd513277dc78d8d08626f3170b99a13ee0ad04bba977147bcaf30358ba25d